src/main/java/Data/ClientParser.java

```diff
@@ -1,137 +1,58 @@
 package Data;
 
-import java.io.*;
-import java.util.ArrayList;
-import java.util.Scanner;
-
-/**
- *
- */
-public class ClientParser
-{
-    public static ArrayList<String> old_file = new ArrayList<>();
-    public static ArrayList<String> new_file = new ArrayList<>();
-    static boolean firstRun = true;
-    static int nOld = 0;
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+import org.apache.commons.io.input.TailerListenerAdapter;
+
+import java.io.File;
+
+
+public class ClientParser {
+
 
     public static volatile String currentZone = "";
     public static volatile String currentLevel = "??";
     public static volatile String playerName = "";
 
     public final static String PATH = Settings.getINSTANCE().getClientTXT().getAbsolutePath();
 
-    public static void coreLoop() throws IOException
-    {
-        nOld = countLines(PATH);
-        StoreText();
-        firstRun = false;
-        //File f = new File(PATH);
-        while (true)
-        {
-            int countLines = countLines(PATH);
-            if (countLines > nOld)
-            {
-                nOld = countLines;
-                StoreText();
-            }
-            try
-            {
-                Thread.sleep(Settings.getPollingRate());
-            } catch (InterruptedException e)
-            {
-                e.printStackTrace();
-            }
-        }
-    }
+    public static final String gameLogFile = PATH + "\\Client.txt";
```

Comment:
Remove: + "\\Client.txt" in order for this to work.

Reply:
oh right I wanted to suggest pointing it at the folder instead of directly at the file.

Reply:
Also not too bad of a way too do it, but I feel like forcing a user to select the Client.txt is a safer way of making sure the Client.txt is selected properly.